### DISCUSSION TITLE
Plowing one direction??

### DISCUSSION THREAD

**Discussion by Aschau11x** [Courseplay/Courseplay_FS22#1750] [Q&A]
Not sure what I'm doing wrong but when setting up CP-fieldwork multiple tools of 2, specifically plowing (so far). The hired workers are only plowing one direction-turning around and just driving back on the path they should be plowing. I have it set to up/down and rows to skip 0. Am I missing something? It's also doing this with just an AI Hired Helper. Is this a bug with giant? The area to the right is what was missed. 
![fsScreen_2022_07_07_22_16_21](https://user-images.githubusercontent.com/108911158/177905451-c6b51a11-2c0a-45e5-9ffe-aae8efdf6f48.png)

**Comment by Tensuko** [comment]
I have no Idea what you try to do... step by step pls...

"multiple tools of 2" -> so you have two vehicles running ?

"I have it set to up/down and rows to skip 0" -> nothing wrong about this.

The tool you have in the screenshot is not an actual plow, must be a mod so it can be a mod Issue!

**Reply by bakkel** [reply]
It's the plow mod you're using! I used the same one and had the same problem. Pick another plow and the problem is solved!

**Reply by Aschau11x** [reply]
Yup, it’s the plow mod. Sorry I meant to post that. Thanks everyone!
